### PR TITLE
feat: commit hook

### PR DESCRIPTION
**Pull request by adevnadia**
Added linting commit hook, npm run commit script, fixed linting errors

### CODE REVIEW

File: .jshintrc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "node": true,
+
+  "curly": true,
+  "latedef": true,
+  "quotmark": true,
+  "undef": true,
+  "unused": true,
+  "trailing": true
+}
```
Comment (bengummer) [inline .jshintrc]: is this file meant to be here? style config currently lives in https://github.com/skatejs/named-slots/blob/master/.eslintrc

File: package.json [hunk 2]
```diff
@@ -8,8 +8,9 @@
     "test": "sk-tests",
     "test/watch": "sk-tests-watch",
     "test/perf": "karma start --single-run --perf",
-    "lint": "eslint -c node_modules/skatejs-build/.eslintrc src",
-    "prepublish": "sk-bundle"
+    "lint": "eslint -c node_modules/skatejs-build/.eslintrc src, test --ext .js,.jsx --format 'node_modules/eslint-friendly-formatter'",
```
Comment (bengummer) [inline package.json]: the `named-slots/.eslintrc` file pulls in `skatejs-build/.eslintrc`, so we probably don't need to directly reference `node_modules/skatejs-build/.eslintrc` here. up to you :)

**Comment by bengummer** [comment]
lgtm (once jshint file question is resolved)

File: .jshintrc (hunk 1 above)
Comment (treshugart) [inline .jshintrc]: +1, this shouldn't need to be here.

File: .jshintignore [hunk 3]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (treshugart) [inline .jshintignore]: This shouldn't be necessary either.

File: package.json (hunk 2 above)
Comment (treshugart) [inline package.json]: This should be implemented as script in skatejs-build, methinks.

**Comment by adevnadia** [comment]
lgtm

**Pull request merged**